kcidb/monitor/output.py

```diff
@@ -174,18 +177,24 @@ def render(self):
             self.obj.get_type().name: self.obj,
         }
         subject = TEMPLATE_ENV.from_string(self.message.subject).render(ctx)
-        subject_extra_bytes = \
-            len(subject.encode()) - NOTIFICATION_MESSAGE_SUBJECT_MAX_LEN
-        assert subject_extra_bytes <= 0, \
-            f"Subject is {subject_extra_bytes} bytes too long"
+        subject_extra_characters = \
+            len(subject) - NOTIFICATION_MESSAGE_SUBJECT_MAX_LEN
+        if subject_extra_characters > 0:
+            subject = subject[:NOTIFICATION_MESSAGE_SUBJECT_MAX_LEN-1]
+            subject += "\u2702"
```

Comment:
If you just put the literal character in here, instead of the escape sequence, the code will fit into one line. Also, please put spaces around that minus character. Something like this:
```suggestion
            subject = subject[:NOTIFICATION_MESSAGE_SUBJECT_MAX_LEN - 1] + "✂️"
```
It's also easier to understand what's going on, if it's the character directly.

BTW, your code is adding the [scissors character](https://codepoints.net/U+2702) (`\u2702`), not the [scissors emoji](https://emojipedia.org/emoji/%E2%9C%82/) (`\u2702\ufe0f`), which is the same character, but followed by emoji [variation selector](https://en.wikipedia.org/wiki/Variation_Selectors_(Unicode_block)) (`\ufe0f`). GitHub is rendering them the same in comments for some reason, but they are not. Is that on purpose?

Same goes for the body, of course.

Reply:
No, it is not 
I will fix it